checkstyle-tester/checks-nonjavadoc-error.xml

```diff
@@ -138,7 +138,6 @@
       <property name="allowSamelineSingleParameterlessAnnotation" value="false"/>
     </module>
     <module name="AnnotationLocation">
-      <property name="tokens" value="PARAMETER_DEF"/>
```

Comment:
I am ok to remove such tokentype, ....
but lets use here all tokens - https://checkstyle.org/config_annotation.html#AnnotationLocation (except for PARAMETER_DEF)

whole reason of this config it use all possible tokens/modes and detect unexpected exceptions of massive code.

@rnveach , does it make sense ?

Reply:
@romani I agree.
Since we are updating to use all tokens, we should do the same with the other `AnnotationLocation` config right above it.

Reply:
done